HelpMyStreetFE/HelpMyStreetFE/src/sass/components/home/_street-check.scss

```diff
@@ -0,0 +1,91 @@
+﻿.street-check {
+    @media screen and (min-width: $med-screen) {
+        padding-left: 273px !important;
```

Comment:
Is this for making the content central? Could we use flexbox instead?

Reply:
I didn't write this, I just moved it into a media query. I haven't done any styling related to the desktop sizes just relocated it. I will have a look at it today.